packages/sw-lib/src/lib/router.js

```diff
@@ -87,10 +64,11 @@ class RouterWrapper {
    *    The only gotcha with this is that it will only capture URL's on your
    *    origin.
    * 1. A regex that will be tested against request URL's.
-   * 1. A [Route]{@link module:sw-routing.Route} instance.
-   * @param {function|Handler} handler The handler is ignored if you pass in
-   * a Route object, otherwise it's required. The handler will be called when
-   * the route is caught by the capture criteria.
+   * 1. A [Route]{@link module:sw-lib.SWLib#Route} instance.
+   * @param {function|Handler} handler The handler argument is ignored if
+   * you pass in a Route object, otherwise it's required. The handler
+   * will be called when the route is caught by the capture criteria.
+   * If required, provide a function or a runtime caching strategy.
```

Comment:
Always start with the description. In this case start with "Called when the...".